spec/models/division_spec.rb

```diff
@@ -83,4 +83,45 @@
       expect(division_3.short_name).to eq('-iamauuid2018')
     end
   end
+
+  describe '#qb_division' do
```

Comment:
This 'bubble up' behavior is used so much that I thought it needed a model spec.